images/docs-builder/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM registry.redhat.io/ubi8
```

Comment:
We used to use `FROM registry.redhat.io/ubi8/ruby-25`, is this not good enough for your needs?

Reply:
we also need Nodejs, the link checking tool is a nodejs script tool, so need to install with npm, it'd probably be possible to install nodejs on the Ruby image, but i went with a generic base image.